src/plugin.js

```diff
@@ -500,8 +481,34 @@ class VR extends Plugin {
 
     this.renderedCanvas = this.renderer.domElement;
 
-    this.renderedCanvas.style.width = 'inherit';
-    this.renderedCanvas.style.height = 'inherit';
+    const debounce = function(fn, wait) {
+      let timeout;
+
+      return function(...args) {
+        // reset the timer
+        if (timeout) {
+          window.clearTimeout(timeout);
+        }
+        timeout = window.setTimeout(() => fn.apply(undefined, args), wait);
+      };
+    };
+    // we use this to stop webvr-polyfill from making the canvas take up more
+    // space then the video element
+    const setInherit = debounce((mut) => {
```

Comment:
A hacky work-around for webvr-polyfills canvas style changes that we don't want.